### ISSUE TITLE
Invalid record in a broker statement

### ISSUE THREAD

**Issue by bolshakov** [KonishchevDmitry/investments#81]
I generated a custom broker statement according to the provided instructions and get the following errors generating tax report:

E: Error while reading "~/Documents/Interactive Brokers/Statements/20220103_20221230.csv" broker statement: Invalid record: ("Credit Interest Details", "Headers", "Currency", "Value Date", "Tier Break", "Rate (%)", "Security Principal", "Futures Principal", "Total Principal", "Security Interest", "Futures Interest", "Total Interest", "Code") (6800 line).

This is the link 6800 with two context lines - one line before and one line after

```
6799: Deposits & Withdrawals,Data,Total,,,12340,
6800: Credit Interest Details,Headers,Currency,Value Date,Tier Break,Rate (%),Security Principal,Futures Principal,Total 6801: Principal,Security Interest,Futures Interest,Total Interest,Code
Credit Interest Details,Data,USD,2022-05-05,II,0.2911887,744.49,0,744.49,0.01,0,0.01,Po
```

**Comment by KonishchevDmitry** [comment]
The third line in your paste looks joined with the second in some strange way. Do you mean the following:

```
6799: Deposits & Withdrawals,Data,Total,,,12340,
6800: Credit Interest Details,Headers,Currency,Value Date,Tier Break,Rate (%),Security Principal,Futures Principal,Total Principal,Security Interest,Futures Interest,Total Interest,Code
6801: Credit Interest Details,Data,USD,2022-05-05,II,0.2911887,744.49,0,744.49,0.01,0,0.01,Po
```
?

The problem in these lines is that we have `,Headers,` instead of `,Header,` which I haven't seen before. Is this the only line with `Headers` instead of `Header`? Does it parse if you change `Headers` to `Header`? Have you edited this file? Could this be your typo?

**Comment by bolshakov** [comment]
> The third line in your paste looks joined with the second in some strange way. Do you mean the following:
Yes, right. 

I haven't edited the source file, and this is the only line with "Headers" instead of "Header" in this file. After replacing it to "Header" I have another issue:

```
E: Unable to find origin operations for the following taxes:
* 10.02.2022: LQD
* 10.02.2022: AGG
* 10.02.2022: GOVT
* 09.02.2022: PLW

Probably manual tax remapping rules are required (see http://bit.ly/investments-ib-tax-remapping).
```

**Comment by KonishchevDmitry** [comment]
Have you managed to solve tax remapping issue using instructions from http://bit.ly/investments-ib-tax-remapping?

**Comment by bolshakov** [comment]
It was solved using tax remapping. By the way, there is one line with "Headers" instead of "Header" in statements generated for 2022 and 2023 years in each file.

**Comment by bolshakov** [comment]
Thank you for your support ❤️

**Comment by KonishchevDmitry** [comment]
Fixed in v5.4.1.

**Comment by bolshakov** [comment]
Thanks a bunch 🙇
